src/main/java/bartenderbob/Ui.java

```diff
@@ -0,0 +1,75 @@
+package bartenderbob;
```

Comment:
I like that you keep all the error messages in the Ui class, thus making a good use of abstraction and encapsulation.